composer.json

```diff
@@ -239,7 +239,10 @@
             "Contao\\NewBundle\\": "core-bundle/tests/Fixtures/vendor/contao/new-bundle/src/",
             "Contao\\NewsBundle\\Tests\\": "news-bundle/tests/",
             "Contao\\TestBundle\\": "core-bundle/tests/Fixtures/vendor/contao/test-bundle/"
-        }
+        },
+        "classmap": [
+            "core-bundle/tests/Fixtures/src/Resources/contao/"
```

Comment:
Why is this needed?

Reply:
For these test classes:

* https://github.com/contao/contao/pull/1909/files#diff-b3a1cc5e20c81dcf50526e017b371a318c9e28b9dbdb36b47bb79440e8384e2b
* https://github.com/contao/contao/pull/1909/files#diff-326722f2b7af2323695401de1c20e2bdf8100e4ed9bfe98852b1e054c2d85ff2

which are there to test the extraction of the defined default template via the reflection API.